README.md

```diff
@@ -243,6 +243,20 @@ flavor:
 * `INSTANCE_MYSQL_URL` If using an external mysql database, set its url here
 * `INSTANCE_MONGO_URL` If using an external mongo database, set its url here
 
+### External SMTP service settings
+
+If you want to use an external SMTP service for sending email from app servers,
+set the following configuration variables:
+
+* `INSTANCE_SMTP_RELAY_HOST` External SMTP host
+* `INSTANCE_SMTP_RELAY_PORT` External SMTP port
+* `INSTANCE_SMTP_RELAY_USERNAME`: External SMTP provider username
+* `INSTANCE_SMTP_RELAY_PASSWORD`: External SMTP provider password
+* `INSTANCE_SMTP_RELAY_SENDER_DOMAIN`: When using external SMTP provider, email
```

Comment:
@mtyaka I think it's awesome that you started putting colons after names of settings! This has been bugging me for a while :) It's a small thing, but IMHO it does improve readability. If you could do that consistently for all settings, that would be great! (If you don't have time, that's fine as well -- just make sure to be consistent within this section in that case.)